BUILDING.md

```diff
@@ -45,6 +45,7 @@ Once you have Rust installed, you will need the following additional dependencie
 + `ld`
 + `grub-mkrescue` & `xorriso`
 + `qemu` for running the kernel under emulation
++ `mtools`
```

Comment:
Depending on whether we need `mtools` on all OSes or not, this might eventually end up in the sections on specific platforms. This is good for now though.